src/Repository/UserRepository.php

```diff
@@ -68,4 +70,27 @@ public function getApplicantsForActivity(ApplicantsListSort $applicantsListSort,
 
         return $queryBuilder;
     }
+
+    /**
+     * @param UserListFilter $userListFilter
+     * @param UserListSort $userListSort
+     * @return QueryBuilder
+     */
+    public function getUserList(
+        UserListFilter $userListFilter,
+        UserListSort $userListSort
+    ): QueryBuilder {
+        $queryBuilder = $this->createQueryBuilder('user');
+        $queryBuilder
+            ->select('user');
+        if ($userListFilter->technology !== null) {
```

Comment:
Use `if (!empty($userListFilter->technology))`. If someone will do a request `...?filter[technology]=&...` (note the missing value for technology) you will get a database error `unexpected closing parenthesis in "technology IN ()"`